x-pack/plugins/actions/server/routes/delete.ts

```diff
@@ -5,18 +5,12 @@
  * 2.0.
  */
 
```

Comment:
can we move this into `x-pack/plugins/actions/server/routes/connector/` into a `delete` folder like the other versioned routes?